reference/tools/meson/meson.rst

```diff
@@ -0,0 +1,32 @@
+.. _conan_tools_meson_meson:
+
+Meson
+======
+
+The ``Meson()`` build helper is intended to be used in the ``build()`` and ``package()`` methods, to call Meson commands automatically
+when a package is being built directly by Conan (create, install).
```

Comment:
```suggestion
The ``Meson()`` build helper is intended to be used in the ``build()`` and ``package()`` methods, to call Meson commands automatically.
```
I would stop the sentence there or add other commands that can call the build or package methods like `build` or `export-pkg`, but I don't think that's necessary.